dts/arm/nxp/nxp_rt1060.dtsi

```diff
@@ -43,6 +45,13 @@
 				interrupt-names = "IEEE1588_TMR";
 			};
 		};
+
+		flexcan3: can@401d8000 {
+			compatible = "nxp,canfd-flexcan";
+			bus-speed-data = <2000000>;
```

Comment:
`bus-speed` should not be set on the SoC level. Also, I believe you're missing `sjw` and `sample-point` properties?

Reply:
Removed the `bus_speed`.